app/views/content/shared/block-promos/_adviser_apply.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="row">
+  <section class="container">
+    <section class="image-blocks">
+      <%= render CallsToAction::AdviserBlockComponent.new %>
```

Comment:
I think we should alter the text on this block to make it application-specific - eg `Maximise your chances of success.....`? We really want people at this point to get an adviser!